experimental/report/testdata/i18n.yaml.color.txt

```diff
@@ -1,8 +1,8 @@
 ⟨b.red⟩error: emoji, CJK, bidi⟨reset⟩
 ⟨blu⟩  --> foo.proto:5:9
    | 
-⟨blu⟩ 5 | ⟨reset⟩message 🐈‍⬛ {
-⟨blu⟩   | ⟨reset⟩⟨reset⟩        ⟨b.red⟩^^⟨reset⟩ ⟨b.red⟩
+⟨blu⟩ 5 | ⟨reset⟩message 🐈<U+200D>⬛ {
+⟨blu⟩   | ⟨reset⟩⟨reset⟩        ⟨b.red⟩^^^^^^^^^^^^⟨reset⟩ ⟨b.red⟩
```

Comment:
You did catch this, and I pointed out that this was caused by a request you made to simplify some of the formatting code, and we decided to leave it as-is because it's harmless.